inject.js

```diff
@@ -29,6 +27,10 @@ module.exports = function (blocks, frame, codec, file, cache) {
     })
   })
 
+  var isDeleted = (b) => Buffer.isBuffer(b) === true && b.equals(Buffer.alloc(b.length)) === true
```

Comment:
Is the === true at the end really necessary? Seems a bit overly defensive? Also I'm a whitespace nazi, I think the spacing after this line would be better left out, so that the two functions align better. This is my only comment, the commit looks good otherwise :+1:

Reply:
Great suggestions, thanks!